.github/workflows/ci-image-checks.yml

```
@@ -155,7 +155,7 @@ jobs:
           platform: ${{ inputs.platform }}
           save-cache: false
       - name: "Static checks"
-        run: prek --all-files --show-diff-on-failure --color always
+        run: prek cache clean && prek --all-files --show-diff-on-failure --color always
```

```suggestion
        run: prek --all-files --show-diff-on-failure --color always --refresh
```
I think this should also do the same

---

Seems to have worked, the rst-backticks hook is passing. 
Do you want to try again with `--refresh` or just stick with what you have now?

---

Make sense! It looks cleaner